vic5_manuscript.tex

```diff
@@ -177,7 +177,7 @@ \section{VIC Infrastructure}
     \end{itemize}
     The implementation details for each of these topics are discussed in the following sections.
 
-    A practical motivating factor in the development of VIC-5 was to enable coupling of VIC within the Regional Arctic System Model \citep[RASM;][]{Hamman_2016a}, a regional implementation of the Community Earth System Model \citep[CESM;][]{Hurrell_2013}, while maintaining usability as a stand-alone model. In RASM, VIC-5 is directly coupled to CESM's flux coupler \citep[CPL7;][]{Craig_2012} via the Model Coupling Toolkit interface \citep[MCT;][]{Larson_2005}. As implemented within RASM, this interface allows VIC to be coupled to the Weather Research and Forecasting (WRF) atmospheric model \citep{Skamarock_2008} and the RVIC streamflow routing model \citep{Hamman_2017a} components, while maintaining the legacy and stand-alone model infrastructure, which have a large existing user base. The successful coupling of VIC-5 within RASM is a key demonstration of the benefits of the source code refactor. The new infrastructure will facilitate VIC to be coupled more readily to other modeling systems in the future.
+    A practical motivating factor in the development of VIC-5 was to enable coupling of VIC within the Regional Arctic System Model \citep[RASM;][]{Hamman_2016a}, a regional implementation of the Community Earth System Model \citep[CESM;][]{Hurrell_2013}, while maintaining usability as a stand-alone model. In RASM, VIC-5 is directly coupled to CESM's flux coupler \citep[CPL7;][]{Craig_2012} via the Model Coupling Toolkit interface \citep[MCT;][]{Larson_2005}. As implemented within RASM, this interface allows VIC to be coupled to the Weather Research and Forecasting (WRF) atmospheric model \citep{Skamarock_2008} and the RVIC (short for routing-VIC) streamflow routing model \citep{Hamman_2017a} components, while maintaining the legacy and stand-alone model infrastructure, which have a large existing user base. The successful coupling of VIC-5 within RASM is a key demonstration of the benefits of the source code refactor. The new infrastructure will facilitate VIC to be coupled more readily to other modeling systems in the future.

```

Comment:
maybe "and the routing-VIC (RVIC)"

Reply:
Nope - RVIC is not an acronym, but simply the name of the new routing model. Since it never was an acronym I don't want to describe it here as one simply to satisfy a reviewer.

Reply:
I've removed the blip: `(short for routing-VIC)`